packages/eslint-plugin/src/rules/use-lifecycle-interface.ts

```diff
@@ -1,4 +1,7 @@
-import { ASTUtils, toPattern } from '@angular-eslint/utils';
+import {
+  ASTUtils, toPattern,
```

Comment:
You will need to run `yarn format` and commit the result to fix the formatting issues (which is why CI is red)